python/interpret_community/explanation/explanation.py

```diff
@@ -522,6 +522,9 @@ def _perf_dict(self, y, y_hat, i):
             di["residual"] = int(y[i] == y_hat[i])
         else:
             di["residual"] = y[i] - y_hat[i]
+        # TODO: for classification case these should be the probabilities
```

Comment:
Might want to create a task for this if you haven't already. (Does not need to be linked in GitHub, obviously)

Reply:
done, created issue:
https://github.com/interpretml/interpret-community/issues/446